private/buf/cmd/buf/command/beta/convert/convert.go

```diff
@@ -152,6 +156,9 @@ func run(
 	if err != nil {
 		return err
 	}
+	if wkpath := wktToPath(flags.Type); input == "" && wkpath != "" {
```

Comment:
I believe this breaks the precedence rules for WKTs in input vs the builtins.

Also see our style guide, specifically the section on descriptive names, i.e. `wellKnownPath` would be correct here if this code persisted.

Reply:
If the input is unspecified it will try to resolve the well known type, if not it will use the input argument. 

This means that calls like previously mentioned `buf beta convert google/protobuf/timestamp.proto --type google.type.Duration` will fail because `google/protobuf/timestamp.proto` doesn't contain `google.type.Duration`